specification/security/resource-manager/Microsoft.Security/preview/2023-02-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1494 @@
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/securityConnectors": {
+      "get": {
+        "x-ms-examples": {
+          "List all security connectors of a specified subscription": {
+            "$ref": "./examples/SecurityConnectors/GetSecurityConnectorsSubscription_example.json"
+          }
+        },
+        "tags": [
+          "SecurityConnectors"
+        ],
+        "description": "Lists all the security connectors in the specified subscription. Use the 'nextLink' property in the response to get the next page of security connectors for the specified subscription.",
+        "operationId": "SecurityConnectors_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityConnectorsList"
+            }
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/securityConnectors": {
+      "get": {
+        "x-ms-examples": {
+          "List all security connectors of a specified resource group": {
+            "$ref": "./examples/SecurityConnectors/GetSecurityConnectorsResourceGroup_example.json"
+          }
+        },
+        "tags": [
+          "SecurityConnectors"
+        ],
+        "description": "Lists all the security connectors in the specified resource group. Use the 'nextLink' property in the response to get the next page of security connectors for the specified resource group.",
+        "operationId": "SecurityConnectors_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityConnectorsList"
+            }
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
```

Comment:
>"$ref": "../../../common/v1/types.json#/definitions/CloudError" [](http://example.com/codeflow?start=14&length=63)

v5 is latest, how about updating?

Reply:
good point 
need to take it with the owner of the repo 
since at our repo under common we only have v1